invisible_cities/io/dst_io.py

```diff
@@ -58,9 +61,16 @@ def _store_pandas_as_tables(h5out : tb.file.File, df : pd.DataFrame, group_name
     table = getattr(group, table_name)
     if not np.array_equal(df.columns, table.colnames):
         raise TableMismatch(f'dataframe differs from already existing table structure')
-    for indx in df.index:
-        tablerow = table.row
-        for colname in table.colnames:
-            tablerow[colname] = df.at[indx, colname]
-        tablerow.append()
+
+    arr = df.to_records(index=False)
+    #hack to transform object numpy dtype to string
+    def _cast_type(dtype : np.dtype):
```

Comment:
I understand where the underscore comes from, but since it is inside a function it doesn't need to have special syntax :)